CMakeLists.txt

```diff
@@ -59,12 +59,17 @@ set_property(CACHE CMAKE_BUILD_TYPE PROPERTY
   STRINGS "Debug" "Dev" "Release" "MinSizeRel" "RelWithDebInfo")
 
 # If not in a git repo (e.g., a tarball) these tokens define the complete
-# version string, else it is combined with the result of `git describe`.
+# version string, else they are combined with the result of `git describe`.
 set(NVIM_VERSION_MAJOR 0)
 set(NVIM_VERSION_MINOR 1)
 set(NVIM_VERSION_PATCH 6)
 set(NVIM_VERSION_PRERELEASE "-dev") # for package maintainers
 
+# API level
+set(NVIM_API_LEVEL 1)         # Bump this after any API change.
+set(NVIM_API_LEVEL_COMPAT 0)  # Adjust this after a _breaking_ API change.
+set(NVIM_API_PRERELEASE false)
```

Comment:
shoulnt prerelease be true right now?

i.e. if 0.1.6 is api:1 then we are in api:1 prerelease:true right now